helpers/flags.js

```diff
@@ -1 +1,16 @@
-export const getFlag = (key, defaultValue) => window.D2L?.LP?.Web?.UI?.Flags.Flag(key, defaultValue) ?? defaultValue;
+const flagOverrides = new Map();
+
+export function getFlag(key, defaultValue) {
```

Comment:
@dbatiste FYI here. As discussed in your PR, this could also handle demo pages if we want.